Source: @pullmoll
Upstream: no
Reason: -1ULL is not a signed long long but the result is signed.

--- src/AddressSpace.hpp	2019-05-30 06:40:21.000000000 +0200
+++ src/AddressSpace.hpp	2020-03-04 22:55:16.140731907 +0100
@@ -288,7 +288,7 @@
   } while (byte & 0x80);
   // sign extend negative numbers
   if ((byte & 0x40) != 0)
-    result |= (-1ULL) << bit;
+    result |= (-1LL) << bit;
   addr = (pint_t) p;
   return result;
 }
